src/common/interfaces/request-exception.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface RequestException {
```

Comment:
In my initial pass I realized the new `RateLimitExceededException` was missing `status`. It seemed worth having something like this in order for all exception classes to be roughly shaped the same.

Arguable we should have a base-class instead, but I opted for an `interface` for now since it only adds an additional compile-time check, and we can decide later.